.github/workflows/ci.yml

```diff
@@ -7,8 +7,23 @@ on:
   pull_request:
 
 jobs:
+  pre_commit:
+    name: 💍 pre-commit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: hashicorp/setup-terraform@v2
+      - name: Run
+        run: |
+          python -m venv .venv
+          source .venv/bin/activate
+          pip install pre-commit checkov
+          pre-commit install
+          pre-commit run --all-files --show-diff-on-failure
```

Comment:
thought: should be removed as it is tackled in #580 and not related to the certificate problem.